Migrations/OrderMigrations.cs

```diff
@@ -43,5 +59,19 @@ public class OrderMigrations : DataMigrationImpl {
                 .AddColumn<int>("UserId", column => column.WithDefault(-1)));
             return 3;

```

Comment:
I think I see what you mean. 
You are saying that the `UpdateFrom3()` that I added needs not run on new tenants, so this `UpdateFrom2()` might as well return 4 already. Right?